internal/handlers/handlers.go

```diff
@@ -1 +1,65 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
+		fmt.Fprint(w, http.StatusMethodNotAllowed)
+		return
+	}
+
+	http.ServeFile(w, r, "index.html")
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		fmt.Fprint(w, http.StatusMethodNotAllowed)
+		return
+	}
+
+	file, header, err := r.FormFile("myFile")
+	if err != nil {
+		fmt.Fprint(w, http.StatusInternalServerError)
```

Comment:
Это ошибка пользователя
```suggestion
		fmt.Fprint(w, http.StatusBadRequest)
```